builder/bootloader.py

```diff
@@ -44,6 +44,35 @@ def get_suitable_optiboot_binary(framework_dir, board_config):
     return bootloader_path
 
 
+def get_bootloader_dxcore(framework_dir, board_config):
+    btld = board_config.get("bootloader.class", "")
+    port = board_config.get("bootloader.port", "")
+    entry = board_config.get("bootloader.entrycond", "")
+
+    if not btld:
+        sys.stderr.write("Error: invalid `bootloader.class` in board config!\n")
+        env.Exit(1)
+    if not port:
+        sys.stderr.write("Error: invalid `bootloader.port` in board config!\n")
+        env.Exit(1)
+
+    if entry:
+        bootloader_file = f"{btld}_{port}_{entry}.hex"
```

Comment:
Are you sure this pattern will properly work with the bootloader binaries from the latest 1.5.6? For example `optiboot_32dd14_ser0_alt3_extr_8sec.hex`.

Reply:
I confirm that the example `optiboot_32dd14_ser0_alt3_extr_8sec.hex` is working with the latest version of DxCore.
```ini
[env]
board_bootloader.class = optiboot_32dd14
board_bootloader.port = ser0_alt3
board_bootloader.entrycond = extr_8sec
```

However, the `toolchain-atmelavr@3.70300.220127` does not support the AVR16DDxx and AVR32DDxx. Only AVR64DDxx are supported.